manifest/owo.Nytra/InspectorDelegateCaller/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "InspectorDelegateCaller",
+	"id": "net.eia485 / Nytra.InspectorDelegateCaller",
```

Comment:
Consider changing the id to be under your author id since it is your fork

Reply:
Can then also add them with `additionalAuthors`

Reply:
Would I also need to change the HarmonyID in the mod? Currently it is "net.eia485 / Nytra.InspectorDelegateCaller" but would I need to change it to my own?

Reply:
if you change one, you would change the other

Reply:
Is that a new thing because I don't think the HarmonyID needed to match the manifest ID before, unless I'm mistaken?

Reply:
They should match unless there reason against doing so. Primarily that would be if you down the line change from `com.example` to `net.example` you wouldn't need to update all the past entries but the latest one should be matching that.

Reply:
I have made the requested changes